aegisthus-hadoop/src/main/java/com/netflix/aegisthus/output/AvroOutputFormat.java

```diff
@@ -63,30 +63,27 @@ public void write(BytesWritable keyWritable, RowWritable rowWritable) throws IOE
                         continue;
                     cgmBuilder.add((Column) atom);
                 }
-
-                // Tombstones?
-                if (cgmBuilder.groups().size() == 0)
-                    return;
```

Comment:
I don't think so, because iteration on empty groups field should give the same result